doc/book/admin/monitoring/alerting.rst

```diff
@@ -0,0 +1,418 @@
+.. _monitoring-alerting-page:
+
+===============================================================================
+Alerting
+===============================================================================
+
+You can set up alerts on metrics to get a notification when something went
+wrong. We will use `Prometheus alert rules <https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/>`_
+as an example here. You can get full ``alerts.yml`` file at
+`tarantool/grafana-dashboard GitHub repo <https://github.com/tarantool/grafana-dashboard/tree/master/example_cluster/prometheus/alerts.yml>`_.
+
+.. _monitoring-alerting-tarantool:
+
+-------------------------------------------------------------------------------
+Tarantool metrics
+-------------------------------------------------------------------------------
+
+You can use internal Tarantool metrics to monitor detailed RAM consumption,
+replication state, database engine status, track business logic issues (like
+HTTP 4xx and 5xx responses or low request rate) and external modules statistics
+(like ``CRUD`` errors). Evaluation timeouts, severity
+levels and thresholds (especially ones for business logic) are placed here for
+the sake of example: you may want to increase or decrease them for your
+application. Also, don't forget to set sane rate time ranges based on your
+Prometheus configuration.
+
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Lua memory
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The Lua memory is limited to 2 GB per instance. Monitoring ``tnt_info_memory_lua``
```

Comment:
Is it still the case? AFAIR this was a limitation before GC64, please recheck with devs.